### DISCUSSION TITLE
4.1.9 image?

### DISCUSSION THREAD

**Discussion by alexbn71** [doitandbedone/ispyagentdvr-docker#465] [General]
Is it normal that the "latest" tag is still 4.1.8?

**Comment by doitandbedone** [comment]
4.1.9 was released few hours ago. Just accepted the change. Should see it
soon.

**Comment by alexbn71** [comment]
Sorry but I was in a hurry to get the latest bug fixes that are important to me. Thanks for your valuable contribution
